src/configs/components.js

```diff
@@ -3,7 +3,6 @@ const { flattenComponents } = require('../utils/flatten-components');
 const components = flattenComponents({
     Button: 'button',
     IconButton: 'button',
-    Link: 'a',
     Spinner: 'svg',
```

Comment:
Should we remove this `svg` map? I think this could potentially be flagging the `svg-has-accessible-name` rule.

Reply:
Ohh I guess [Spinner](https://primer.style/react/Spinner) doesn't render with `aria-hidden` or an `aria-label` by default.. But I feel like this should be taken care of at a Primer React level than by each consumer.

Reply:
@khiga8, good call! We could potentially handle it in Primer, but I'm not too sure which route we would take. We could hide it through `aria-hidden`, but there are usages where the only indication of a loading state is through this component. The `a11y-svg-has-accessible-name` violation might help with usages where it's being used without an alternative way to convey the loading state.

Would we want to hide it in all cases regardless? Either route, I definitely think some documentation on this could be added to our [docs for Spinner](https://primer.style/react/Spinner).

Reply:
Hmmmm, perhaps we'd want a default accessible name (which could be overriden by the consumer to be more specific)? So at minimum all spinners have the accessible name, "Loading"...

But I think we would need to do a deeper dive into Spinner to figure out what the exact markup/API should be.

Reply:
Yeah, we could do that! I do think a deeper dive into this component is worth it. In the meantime, would we want to keep the mapping, or do you think it's best to remove it? Right now, I believe this mapping is throwing the most violations out of all the other components in our mapping.

Reply:
Yeah, I think we should remove it for now!

Reply:
Sounds good! I removed it, and will follow up by updating the disables for this component. Will handle this on the component itself!